src/gcp/cloudfunctions.ts

```diff
@@ -20,17 +33,126 @@ interface Operation {
   error?: { code: number; message: string };
 }
 
-export const API_VERSION = "v1";
+export interface HttpsTrigger {
+  // output only
+  url?: string;
```

Comment:
Could use `readonly` for this if we want to get super nitty ([docs](https://www.typescriptlang.org/docs/handbook/classes.html#readonly-modifier)).

Reply:
SGTM